Cargo.toml

```
@@ -49,7 +49,7 @@ hex = { version = "0.4", default-features = false, features = ["alloc"] }
 postcard = { version = "1.0" }
 rand = { version = "0.9" }
 reqwest = "0.12"
-risc0-aggregation = "0.2.1"
+risc0-aggregation = { version = "0.2" }
```

```suggestion
risc0-aggregation = { version = "0.2.2" }
```
Looking at the tag of the guest-set-builder, it seems that is locked to v0.2.2 so we should assure here to at least use version 0.2.2 risc0-aggregator (maybe even fix it to that exact version).